snap-server.cabal

```
@@ -112,7 +112,7 @@ Library
     old-locale                          >= 1.0      && < 1.1,
     snap-core                           >= 1.0      && < 1.1,
     text                                >= 0.11     && < 1.3,
-    time                                >= 1.0      && < 1.11,
+    time                                >= 1.0      && < 1.12,
```

This bound is outdated already
```suggestion
    time                                >= 1.0      && < 1.13,
```